streams/src/main/java/org/apache/kafka/streams/kstream/internals/suppress/FinalResultsSuppressionBuilder.java

```diff
@@ -22,7 +22,7 @@
 import java.time.Duration;
 import java.util.Objects;
 
-public class FinalResultsSuppressionBuilder<K extends Windowed> implements Suppressed<K>, NamedSuppressed {
+public class FinalResultsSuppressionBuilder<K extends Windowed> implements Suppressed<K>, NamedSuppressed<K> {
```

Comment:
nit: sufficient ti extend `NameSuppressed` now

Reply:
It is redundant now, but would it be better to leave them separate?

`NamedSuppressed` is a bandage just until there's some other interface that guarantees a `name()` method, and `NamedSuppressed` only extends `Suppressed` to guarantee it won't be misused for other config classes.

Given both of these, listing them both here means that we can just slice out the `NamedSuppressed` when the time comes.

Not a big deal, though, just sharing my thoughts. I'm happy to make the change if you prefer.

Reply:
It's internal. So should be fine.